Parking_Lot_Reservation/Parking_Lot_Reservation.csproj

```diff
@@ -18,8 +18,4 @@
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.2" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="Controllers\" />
```

Comment:
why

Reply:
Honestly I don't know how this happened.

Reply:
xD